RoundsWithFriends/GameModes/GM_Deathmatch.cs

```diff
@@ -21,6 +22,7 @@ public class GM_Deathmatch : MonoBehaviour
 		private bool isTransitioning;
 		private int playersNeededToStart = 2;
 		private int currentWinningTeamID = -1;
+        private int? timeUntilBattleStart = null;
```

Comment:
Check indentation

Reply:
Fixed in the PR that will replace this one shortly https://github.com/pdcook/RoundsWithFriends/tree/GameModeAbstraction2